### PR TITLE
Added provider anizb.org

### PR DESCRIPTION
**Pull request by p0psicles**
Provider provides an api for backlog and rss for anime shows.

TODO:
- [x] Oops missed the size parsing
- [x] Fix quantifiedCode errors

image

### CODE REVIEW

**Comment by p0psicles** [comment]
@duramato can you review and test this one? just too make sure i didn't miss anything.

File: sickbeard/providers/anizb.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+        self.url = 'https://anizb.org/'
+        self.urls = {
+            'rss': self.url,
+            'api': urljoin(self.url, 'api/?q=')
+        }
+        # Proper Strings
+
+        # Cache
+        self.cache = tvcache.TVCache(self, min_time=15)  # only poll Torrentz every 15 minutes max
```
Comment (duramato) [inline sickbeard/providers/anizb.py]: "only poll Torrentz every 15 minutes max" copy pasta fail lol

**Comment by p0psicles** [comment]
image

**Comment by p0psicles** [comment]
Approved

**Pull request merged**